src/targets/targets.test.ts

```diff
@@ -62,7 +62,7 @@ availableTargets()
               const { convert } = new HTTPSnippet(request);
               const result = convert(targetId, clientId); //?
 
-              expect(result).toStrictEqual(expected);
+              expect((result as string).trim()).toStrictEqual(expected.trim());
```

Comment:
`.editorconfig` adding new lines but the conversion not has made making changes to these fixtures a bit annoying.

Reply:
I know I know I know exactly what you mean.. and I did the same thing originally, but the eventual problem is that whitespace and newlines are truly significant in some cases and (as a design goal) I really want what's in these files to literally match the output - even if they do crazy things.

For example, the `http1.1` targets will get a whole lot less specific.  they even do one of the strangest things of all, which is combine CRLF and LF in the same file.... but ..... that's the spec.

would you mind reverting this?  I totally get that it's annoying, but in my view it's just a fact of life.  I do `Ctrl+P` and `save without formatting` to get around this, personally (in vscode).

Reply:
@dimitropoulos Done